servers/store/src/test/java/org/projectnessie/server/store/TestStoreWorker.java

```diff
@@ -167,7 +178,8 @@ void viewMetadataLocationGlobal() {
                         IcebergMetadataPointer.newBuilder()
                             .setMetadataLocation("metadata-location"))
                     .build()
-                    .toByteString());
+                    .toByteString(),
+            contentAttachmentKeyStream -> Stream.empty());
```

Comment:
nit: define a constant for this empty stream to go together with `ALWAYS_THROWING_ATTACHMENT_CONSUMER`?